.github/workflows/debug-smoke.yml

```diff
@@ -0,0 +1,30 @@
+# Build spike and run a couple of debug tests.
+
+name: Debug Quick Test
+
+# Temporary, so I can test this automation in my draft PR
+#on:
+#  push:
+#    branches:
+#      - master
+#  pull_request:
+#    branches:
+#      - master
+
+on: [push,pull_request]
+
+jobs:
+  test:
+    name: Test debug (Ubuntu)
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        run: sudo xargs apt-get install -y < .github/workflows/apt-packages.txt
+
+      - run: |
+          git clone --depth 1 https://github.com/riscv-software-src/riscv-tests.git
```

Comment:
We might want to check out a known-working commit (which we can periodically update), else changes to riscv-tests risk preventing Spike PRs from being merged.

Reply:
Sorry if this is getting you email spam. I had hoped that marking this as a draft would avoid that.

Let me see if I can get it working first, and then I'll worry about making it nice.

Reply:
no worries!

Reply:
Fixed.